api/observability/tracing/ssh/session.go

```diff
@@ -342,3 +342,47 @@ func (s *Session) CombinedOutput(ctx context.Context, cmd string) ([]byte, error
 	output, err := s.Session.CombinedOutput(cmd)
 	return output, trace.Wrap(err)
 }
+
+// FileTransferRequestApprove sends a "file-transfer-request-response" ssh request
+// The response will contain an Approve bool which will approve or deny a requested file transfer
+func (s *Session) FileTransferRequestResponse(ctx context.Context, req FileTransferResponseReq) error {
+	const request = "file-transfer-request-response"
+	config := tracing.NewConfig(s.wrapper.opts)
+	ctx, span := config.TracerProvider.Tracer(instrumentationName).Start(
```

Comment:
Do we need to set up all of this tracing stuff even though in the end we end up just calling `s.Session.SendRequest` which does that again? I don't really know much about how we do tracing, I'm looking at `SetEnv` which ends up calling `SendRequest` too, but `SetEnv` actually does some extra stuff.

Reply:
`s.Session.SendRequest` doesn't have the tracing as that uses the wrapped `ssh.Session`, but if I instead just used `s.SendRequest`, then yeah, I can ignore the tracing as _that_ `SendRequest` does set it up. Good call

Reply:
Ah, I didn't notice the difference between the two. Still, I'd recommend asking on #dev-teleport if one way of doing this is preferred over the other.